resources/Materials/TestSuite/stdlib/math/transform.mtlx

```diff
@@ -175,6 +175,7 @@
       <input name="scale" type="vector2" value="2.0,2.0" />
       <input name="rotate" type="float" value="1.0" unittype="angle" unit="radian" />
       <input name="offset" type="vector2" value="0.2,0.2" />
+      <input name="operationorder" type="integer" value="0" />
```

Comment:
Since this binding is to the default value, I'd recommend that we omit this added line and just leverage the default.